youthful-piety/modernized/06-chapter-05.adoc

```diff
@@ -547,7 +555,7 @@ it may be more convenient to attend to them.--But,
 ah! little do such disobedient children think,
 that the present time is the only convenient season--their heavenly Father,
 whom they thus slight and neglect, may not visit them again by his good Spirit;
-and in the midst of their play and folly, death may cut them off May all, therefore,
+and in the midst of their play and folly, death may cut them off. May all, therefore,
 of whatever age, hearken to the reproofs of this divine monitor,
 surrender their whole hearts to the Lord, and obey his blessed instructions;
 for this is the way to obtain eternal life.
```

Comment:
Github won't let me put a mark on line 589, but there is actually another testimony that starts there for CHARLES COLEBY, which needs to be separated from the paragraph above, with an asterism. (same with the original) 👍🏻